examples/simple-generator/src/app/ui5.ts

```diff
@@ -39,6 +39,9 @@ export type UI5SupportVersions = {
 
 let ui5VersionSupport: UI5SupportVersions;
 
+/**
+ *
+ */
```

Comment:
Is this a todo?

Reply:
https://github.com/SAP/open-ux-tools/pull/649/commits/0931d94d9973bbdcac725c10ac85a80dff8fb61a

Reply:
Thanks!